tests/Integration/Behaviour/behat.yml

```diff
@@ -6,6 +6,12 @@ default:
             contexts:
                 - Tests\Integration\Behaviour\Features\Context\CommonFeatureContext
                 - Tests\Integration\Behaviour\Features\Context\SqlManagerFeatureContext
+        contact:
```

Comment:
@matks @rokaszygmantas had to include this here instead of the bottom after CLDR and some other suites are launched test database was not reset as expected; For example: ps_lang table had 7 different languages instead of just English.

Reply:
Then it means the database is not properly initialized or clean The order shouldn't matter as the tests are supposed to be independent so what you say is quite worrying as some other context could influence others.. That's not problem as long as it triggers an error ^^ But the risk is having false positive
I thought databases were automatically reset, but it seems it is linked to https://github.com/PrestaShop/PrestaShop/blob/0facef127b6d512cce953c5e2c693705af46fdec/tests/Integration/Behaviour/Features/Context/CommonFeatureContext.php#L65
So I guess you need to add `@reset-database-before-feature` in your feature file Or `@database-scenario` before a scenario (but this one is not used for now)

Reply:
Oh you already have the `@reset-database-before-feature` so it means it doesn't work properly?

Reply:
Yes, the Behat cases did not work properly when contact feature runs at the end. When I added new contact, instead of just only 1 item inside localisedTitles and localisedDescription, I've received 7. One for each language which were created with CLDR feature. 

Not sure why these languages were not cleaned with @reset-database-before-feature or if they were cleared how they reappeared. Also it could be that cache is not disabled properly but I see that it is disabled for the ObjectModel. Maybe object model disable Cache not working properly for Languages?

When DatabaseDump::restoreDb(); happens, I can see in ps_dump.sql that the ps_lang table is dropped though:

```
--
-- Table structure for table `ps_lang`
--

DROP TABLE IF EXISTS `ps_lang`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!40101 SET character_set_client = utf8 */;
CREATE TABLE `ps_lang` (
  `id_lang` int(11) NOT NULL AUTO_INCREMENT,
  `name` varchar(32) COLLATE utf8_unicode_ci NOT NULL,
  `active` tinyint(1) NOT NULL,
  `iso_code` varchar(2) COLLATE utf8_unicode_ci NOT NULL,
  `language_code` varchar(5) COLLATE utf8_unicode_ci NOT NULL,
  `locale` varchar(5) COLLATE utf8_unicode_ci NOT NULL,
  `date_format_lite` varchar(32) COLLATE utf8_unicode_ci NOT NULL,
  `date_format_full` varchar(32) COLLATE utf8_unicode_ci NOT NULL,
  `is_rtl` tinyint(1) NOT NULL,
  PRIMARY KEY (`id_lang`)
) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
/*!40101 SET character_set_client = @saved_cs_client */;
```
So not sure why it happens like that:

```
    Then contact 4 should have the following details:  # Tests\Integration\Behaviour\Features\Context\Domain\ContactFeatureContext::contactShouldHaveTheFollowingDetails()
      | title                     | test service 2      |
      | email_address             | test@prestashop.com |
      | is_message_saving_enabled | true                |
      | description               | test12345           |
      | shop_id_association       | 1                   |
      Failed asserting that two objects are equal.
      --- Expected
      +++ Actual
      @@ @@
       PrestaShop\PrestaShop\Core\Domain\Contact\QueryResult\EditableContact Object (
           'contactId' => 4
           'localisedTitles' => Array (
               1 => 'test service 2'
      +        2 => 'test service 2'
      +        3 => 'test service 2'
      +        4 => 'test service 2'
      +        5 => 'test service 2'
      +        6 => 'test service 2'
      +        7 => 'test service 2'
           )
           'email' => PrestaShop\PrestaShop\Core\Domain\ValueObject\Email Object (...)
           'isMessagesSavingEnabled' => true
           'localisedDescription' => Array (
               1 => 'test12345'
      +        2 => ''
      +        3 => ''
      +        4 => ''
      +        5 => ''
      +        6 => ''
      +        7 => ''
           )
           'shopAssociation' => Array (...)
       )

```
When I query the tesdb at the end I see only one language in the output. So my guess is that object model cache for languages is not disabled properly in this case.

Reply:
I think it's valid to have a contact dedicated behat suite.
About CLDR messing up our tests, we will need to explore this issue too 😋but current solution is relevant to me 👍